test/oauth.test.ts

```diff
@@ -0,0 +1,44 @@
+import type { AxiosRequestConfig } from "axios";
+import axios from "axios";
+import { OAuth } from "../dist/oauth";
+
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const adapter = (config: AxiosRequestConfig): Promise<any> => {
+  return Promise.resolve({
+    data: {
+      method: config.method,
+      path: config.url,
+      data: JSON.parse(config.data),
+    },
+  });
+};
+const oauth = new OAuth(axios.create({ adapter }));
+
+describe("oauth.authenticate", () => {
+  test("session", () => {
+    return expect(
+      oauth.authenticate("DOYoip3rvIMMW5lgItikFK-Ak1CfMsgjuiCyI7uuU94=")
```

Comment:
This looks like the Magic Link 200 token. I know it doesn't matter for these tests because we're just checking string equality, but I think it'd be nice to use the OAuth 200 token here if we have one.

If we do have an OAuth sandbox token, could you also add a test to `sandbox.test.ts` that uses it? I've been using that as a lightweight integration test to ensure that the types are all usable and accurate.

Reply:
Changed it to fake-token until we add a real sandbox token